test/perf/tablet_load_balancing.cc

```diff
@@ -374,7 +364,7 @@ future<results> test_load_balancing_with_many_tables(params p, bool tablet_aware
 
         check_balance();
 
-        rebalance_tablets(e.get_tablet_allocator().local(), stm);
+        rebalance_tablets(e);
```

Comment:
This patch can be sent (and backported) independently of the series, no?

Reply:
Yes